java/rcloadenv/src/main/java/com/google/cloud/rcloadenv/RuntimeConfigLoaderUtil.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
The [latest Google Apache header](https://opensource.google.com/docs/releasing/preparing/#license) does not include "All Rights Reserved."